### PR TITLE
[zh-cn] Updated namespaces.md

### PR DESCRIPTION
**Pull request by gaogao101**
Signed-off-by: Gao Qian <gaoq.jy@fujitsu.com>

### CODE REVIEW

File: content/zh-cn/docs/concepts/overview/working-with-objects/namespaces.md [hunk 1]
```diff
@@ -61,6 +61,47 @@ resources within the same namespace.
...
+<!--
+For a production cluster, consider _not_ using the `default` namespace. Instead, make other namespaces and use those.
+-->
+对于生产集群，请考虑**不要**使用 `default` 名字空间，而是创建其他名字空间来使用。
+{{< /note >}}
+
+<!--
+## Initial namespaces
+-->
+## 初始名字空间
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/overview/working-with-objects/namespaces.md:74]:
```suggestion
## 初始名字空间   {#initial-namespaces}
```

File: content/zh-cn/docs/concepts/overview/working-with-objects/namespaces.md [hunk 2]
```diff
@@ -61,6 +61,47 @@ resources within the same namespace.
...
+`kube-system`
+: The namespace for objects created by the Kubernetes system.
+-->
+Kubernetes 启动时会创建四个初始名字空间：
+
+`default`
+: Kubernetes 包含这个名字空间，由此你无需创建新的名字空间即可开始使用新集群。
+
+`kube-node-lease`
+: 该名字空间用于与各个节点相关的 [Lease（租约）](/zh-cn/docs/reference/kubernetes-api/cluster-resources/lease-v1/) 对象。节点租约允许 kubelet 发送 [心跳](/zh-cn/docs/concepts/architecture/nodes/#heartbeats)，由此控制面能够检测到节点故障。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/overview/working-with-objects/namespaces.md:97]:
```suggestion
: 该名字空间包含用于与各个节点关联的 [Lease（租约）](/zh-cn/docs/reference/kubernetes-api/cluster-resources/lease-v1/) 对象。
  节点租约允许 kubelet 发送[心跳](/zh-cn/docs/concepts/architecture/nodes/#heartbeats)，由此控制面能够检测到节点故障。
```

File: content/zh-cn/docs/concepts/overview/working-with-objects/namespaces.md [hunk 3]
```diff
@@ -61,6 +61,47 @@ resources within the same namespace.
...
+
+`kube-public`
+: *所有*的客户端（包括未经身份验证的客户端）都可以读取该名字空间。该名字空间主要供集群使用，以防某些资源需要在整个集群中可见可读。该名字空间的公共方面只是一种约定而非要求。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/overview/working-with-objects/namespaces.md:100]:
```suggestion
: **所有**的客户端（包括未经身份验证的客户端）都可以读取该名字空间。
  该名字空间主要预留为集群使用，以防某些资源需要在整个集群中可见可读。
  该名字空间的公共属性只是一种约定而非要求。
```

File: content/zh-cn/docs/concepts/overview/working-with-objects/namespaces.md [hunk 4]
```diff
@@ -61,6 +61,47 @@ resources within the same namespace.
...
+
+`kube-system`
+: Kubernetes 系统创建对象所使用的名字空间。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/overview/working-with-objects/namespaces.md:103]:
```suggestion
: 该名字空间用于 Kubernetes 系统创建的对象。
```

File: content/zh-cn/docs/concepts/overview/working-with-objects/namespaces.md [hunk 5]
```diff
@@ -61,6 +61,50 @@ resources within the same namespace.
...
+`kube-public`
+: This namespace is readable by *all* clients (including those not authenticated). This namespace is mostly reserved for cluster usage, in case that some resources should be visible and readable publicly throughout the whole cluster. The public aspect of this namespace is only a convention, not a requirement.
+
+`kube-system`
+: The namespace for objects created by the Kubernetes system.
+-->
+Kubernetes 启动时会创建四个初始名字空间：
+
+`default`
+: Kubernetes 包含这个名字空间，由此你无需创建新的名字空间即可开始使用新集群。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/overview/working-with-objects/namespaces.md:94]:
```suggestion
: Kubernetes 包含这个名字空间，以便于你无需创建新的名字空间即可开始使用新集群。
```

File: content/zh-cn/docs/concepts/overview/working-with-objects/namespaces.md [hunk 6]
```diff
@@ -61,6 +61,50 @@ resources within the same namespace.
...
+
+`kube-node-lease`
+: 该名字空间包含用于与各个节点关联的 [Lease（租约）](/zh-cn/docs/reference/kubernetes-api/cluster-resources/lease-v1/) 对象。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/overview/working-with-objects/namespaces.md:97]:
```suggestion
: 该名字空间包含用于与各个节点关联的 [Lease（租约）](/zh-cn/docs/reference/kubernetes-api/cluster-resources/lease-v1/)对象。
```

File: content/zh-cn/docs/concepts/overview/working-with-objects/namespaces.md [hunk 7]
```diff
@@ -61,6 +61,50 @@ resources within the same namespace.
...
+  节点租约允许 kubelet 发送[心跳](/zh-cn/docs/concepts/architecture/nodes/#heartbeats)，由此控制面能够检测到节点故障。
+
+`kube-public`
+: **所有**的客户端（包括未经身份验证的客户端）都可以读取该名字空间。
+  该名字空间主要预留为集群使用，以防某些资源需要在整个集群中可见可读。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/overview/working-with-objects/namespaces.md:102]:
```suggestion
  该名字空间主要预留为集群使用，以便某些资源需要在整个集群中可见可读。
```

**Comment by windsonsea** [comment]
/lgtm
thanks.

**Pull request merged**